drivers/gpu/drm/rp1/rp1-vec/rp1_vec_cfg.c

```diff
@@ -484,17 +166,18 @@ void rp1vec_vidout_setup(struct rp1_vec *vec)
 	 * clashing requirements for PLL_VIDEO, and potentially for VDAC).
 	 * We therefore leave DPI memories powered down.
 	 */
-	CFG_WRITE(VIDEO_OUT_CFG_MEM_PD, VIDEO_OUT_CFG_MEM_PD_DPI_BITS);
-	CFG_WRITE(VIDEO_OUT_CFG_TEST_OVERRIDE, 0x00000000);
+	CFG_WRITE(VIDEO_OUT_CFG_MEM_PD, VIDEO_OUT_CFG_MEM_PD_DPI);
+	CFG_WRITE(VIDEO_OUT_CFG_TEST_OVERRIDE, 0);
 
-	/* DPI->Pads; VEC->VDAC */
-	CFG_WRITE(VIDEO_OUT_CFG_SEL, VIDEO_OUT_CFG_SEL_VDAC_MUX_BITS);
+	/* VEC->Pads (GPIOs 11:4 with clock on GPIO0); VEC->VDAC */
+	CFG_WRITE(VIDEO_OUT_CFG_SEL,
+		  VIDEO_OUT_CFG_SEL_VDAC_MUX | VIDEO_OUT_CFG_SEL_PAD_MUX);
```

Comment:
^ The register style change somewhat obscures it; the functional change is this line only.